META-INF/MANIFEST.MF

```diff
@@ -1,5 +0,0 @@
-Manifest-Version: 1.0

```

Comment:
I had imported a jar file instead of referencing it and thought this is coming from there. So, mistake